cylc/flow/task_outputs.py

```diff
@@ -601,7 +601,7 @@ def color_wrap(string, is_complete):
     @staticmethod
     def is_valid_std_name(name: str) -> bool:
         """Check name is a valid standard output name."""
-        return name in SORT_ORDERS
+        return name in TASK_OUTPUTS
```

Comment:
Slightly surprised by this change.

We were able to use `:finished` outputs before, why is the change only needed now?

Reply:
Terminals doesn't do this processing anywhere else, other use cases for this do.

Reply:
```suggestion
        return name in SORT_ORDERS
```